app/bundles/DashboardBundle/Event/WidgetDetailEvent.php

```diff
@@ -149,26 +175,41 @@ public function getTemplate()
 
     /**
      * Set the widget template data.
+     *
+     * @throws \Psr\Cache\InvalidArgumentException
      */
-    public function setTemplateData(array $templateData, $skipCache = false): void
+    public function setTemplateData(array $templateData, ?bool $skipCache = false): bool
```

Comment:
We cannot change the method signature. Pease add the return types as method annotations. Is it necessary to change the return type? It's not used anywhere as far as I can see.

If it's necessary to change the types, then please create a new method for it and mark this one as deprecated.

Reply:
This function was returning boolean value in return statement but now I have verified that response of this function is not used anywhere so I have reverted again return type to void.